cinema_service/settings.py

```diff
@@ -42,10 +41,20 @@
     "django.contrib.staticfiles",
     "rest_framework",
     "debug_toolbar",
+    'django_filters',
     "cinema",
     "user",
 ]
 
+
+# REST_FRAMEWORK = {
```

Comment:
it is not very clear why to comment the code, I think it should be removed
this is not a good practice

Reply:
fixed! Thanks